codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/utils/IOUtils.java

```diff
@@ -133,6 +136,21 @@ public static String evalTemplate(
     return evalTemplate(template, context);
   }
 
+  /**
```

Comment:
Am I missing something or is this unused? (And duplicates `safeEvalPathTemplate`?

Reply:
Good catch, part of an earlier refactoring attempt to fix the problem. Deleted.